### ISSUE TITLE
Problem in customFunction

### ISSUE THREAD

**Issue by ferdinandocecchini** [fayaz07/animated_splash#11]
I noted that the customFunction is executed repeatedly.
Is this an expected behavior?
I set a duration of 5 seconds and during this time the following simple customFunction get executed 3 times (this function is just to test the outputAndHome parameter):
<pre>
Function duringSplash = () {
    int rnd = Random().nextInt(10);
    print('rnd $rnd');
    return rnd < 5 ? 1 : 2;
  };
</pre>
Into the console you can see 3 times the value of rnd variable.

**Comment by fayaz07** [comment]
No, it should execute only once. But yeah I observed initState getting called multiple times in flutter app while using hot reload. Did you see that behavior in a normal run without hot reload.

**Comment by ferdinandocecchini** [comment]
Yes, it happens even with a normal start.
I've just launched Android Studio, started the IOS Simulator and click on Run button and this is the result:
![image](https://user-images.githubusercontent.com/5355594/79066066-334f9600-7cb5-11ea-9614-4f2da753b19c.png)

**Comment by fayaz07** [comment]
Hi @ferdinandocecchini , yeah I actually gone through the issue and I had to make lot of changes there. I will publish the changes in the next version

**Comment by ferdinandocecchini** [comment]
I made some small changes to the code like:
- custom background color
- The Image parameter can be any type of image (.asset, .networkimage, ...)
- you can enter multiple lines of text on the home screen and set the distance from the image
If you wish, I can send you the dart file so that we can integrate this simple improvements into the next version (if you think they can be interesting!)
